chromatic/rainbows/actions/remove_trends.py

```diff
@@ -0,0 +1,120 @@
+from ...imports import *
+
+
+def remove_trends(
+    self, method="median_filter", **kw
+):  # model=None,win_length=None,polyorder=None,butter_cutoff=None,butter_fs=None, butter_order=9,
+    """
+    A quick tool to approximately remove astrophysical signals,
+    usually by some combination of smoothing/filtering.
+
+    Parameters
+    ----------
+    method : str
+        What method should be used to make an approximate model
+        for smooth trends that will then be subtracted off?
+
+        `differences` will do an extremely rough filtering
+        of replacing the fluxes with their first differences.
+        Trends that are smooth relative to the noise will
+        be removed this way, but sharp features will remain.
+        Required keywords:
+            None.
+
+        `median_filter` is a wrapper for scipy.signal.median_filter.
+        It smoothes each data point to the median of its surrounding
+        points in time and/or wavelength. Required keywords:
+            `size` = centered on each point, what shape rectangle
+            should be used to select surrounding points for median?
+            The dimensions are (nwavelengths, ntimes), so `size=(3,7)`
```

Comment:
It took me a while to figure out which dimension was which, so I thought it might help to include an example of what the numbers mean in here!